osu.Game.Tests/Visual/TestCaseBeatmapInfoWedge.cs

```diff
@@ -126,6 +127,26 @@ private void testNullBeatmap()
             AddAssert("check no infolabels", () => !infoWedge.Info.InfoLabelContainer.Children.Any());
         }
 
+        private void testAsyncLoading()
+        {
+            RulesetInfo firstRuleset = rulesets.AvailableRulesets.ElementAt(0); // Should be osu standard
+            RulesetInfo secondRuleset = rulesets.AvailableRulesets.ElementAt(1); // Should be taiko
```

Comment:
`osu.Game.Tests` has access to the rulesets. Can just specify `new OsuRuleset().RulesetInfo` / `new TaikoRuleset().RulesetInfo`.